kpconv_torch/datasets/S3DIS.py

```diff
@@ -76,25 +80,18 @@ def __init__(
         config.num_classes = self.num_classes - len(self.ignored_labels)
         config.dataset_task = self.dataset_task
 
-        # Parameters from config
-        self.config = config
-
         # Training or test set
         if split not in ["training", "validation", "test", "ERF", "all"]:
             raise ValueError("Unknown set for S3DIS data: ", split)
```

Comment:
I think this can be added to `PointCloudDataset` (to be checked).

Reply:
The principle is ok for me. Nevertheless, this check isn't made in the other datasets. Are the split options supposed to be the same for all datasets or can there be a difference?

Reply:
It is not made because [that's me who added the check recently](https://github.com/delhomer/kpconv-torch/commit/eab212ace562d2bca8f57076f2f937f387a73717#diff-4262f3c01dd16e05b34593b92ec4653b90afcb0e760a822693df06761e74d70eR73). (Quick win so as to use the S3DIS dataset...)

In my opinion, we can do the assumption that the splits are equal whatever the dataset.